app/src/main/java/org/sopt/pingle/presentation/ui/main/plan/PlanSummaryConfirmationFragment.kt

```diff
@@ -0,0 +1,38 @@
+package org.sopt.pingle.presentation.ui.main.plan
+
+import android.os.Bundle
+import android.view.View
+import androidx.fragment.app.activityViewModels
+import org.sopt.pingle.R
+import org.sopt.pingle.databinding.FragmentPlanSummaryConfirmationBinding
+import org.sopt.pingle.presentation.type.CategoryType
+import org.sopt.pingle.util.base.BindingFragment
+import org.sopt.pingle.util.fragment.colorOf
+
+class PlanSummaryConfirmationFragment :
+    BindingFragment<FragmentPlanSummaryConfirmationBinding>(R.layout.fragment_plan_summary_confirmation) {
+    private val viewModel by activityViewModels<PlanViewModel>()
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+
+        initLayout()
+    }
+
+    private fun initLayout() {
+        val category: CategoryType? = viewModel.selectedCategory.value
+        if (category != null) {
+            with(binding) {
+                badgePlanSummaryConfirmationCategory.setBadgeCategoryType(category)
+                tvPlanSummaryConfirmationName.setTextColor(colorOf((category.textColor)))
+                tvPlanSummaryConfirmationName.text = viewModel.planTitle.value
+                tvPlanSummaryConfirmationOwnerName.text = "개최자"
+                tvPlanSummaryConfirmationCalenderDetail.text =
+                    viewModel.planDate.value + "\n" + makeTimeClean(viewModel.startTime.value) + " ~ " + makeTimeClean(viewModel.endTime.value)
+                tvPlanSummaryConfirmationMapDetail.text = viewModel.selectedLocation.value?.location
+                tvPlanSummaryConfirmationRecruitmentDetail.text = getString(R.string.plan_summary_confirmation_recruitment_number, viewModel.selectedRecruitment.value)
+            }
+        }
+    }
+
+    private fun makeTimeClean(time: String): String = time.substring(0, 5)
```

Comment:
```suggestion
    private fun convertWithoutSecondFormatHours(time: String): String = time.substring(0, 5)
```

이런 식으로 네이밍하는 게 좋을 것 같고 0이랑 5는 상수화 해주세요

Reply:
뭔가 직관적이지 않은 것 같아서 convertDateFormat, convertTimeFormat으로 수정했슴둥